tools/harness-simulation/posix/sniffer_sim/proto/sniffer.proto

```diff
@@ -41,6 +44,13 @@ message StartResponse {
     Status status = 1;
 }
 
+message TransferFileRequest {
```

Comment:
```suggestion
message TransferPcapRequest {
```
nit

Reply:
I will use `TransferPcapngRequest` instead as `tshark` generates output in PCAPNG format in default.